docker/docker-compose/README.md

```diff
@@ -9,10 +9,11 @@ Requirements:
 
 ### Deploy
 
-Manually copy SQL files from `inlong-manager/sql` to the `docker/compose/sql` directory.
+Manually copy SQL files from `inlong-manager/sql` and `inlong-audit/sql` to the `docker/docker-compose/sql` directory.
 
 ```shell
-cp inlong-manager/sql/apache_inlong_manager.sql  docker/docker-compose/sql
+cp inlong-manager/sql/apache_inlong_manager.sql docker/docker-compose/sql
+cp inlong-audit/sql/apache_inlong_audit.sql docker/docker-compose/sql
 ```
```

Comment:
@dockerzhang It seems that `apache_inlong_manager.sql` is not in the `inlong-manager/sql` directory. Which path should be changed to?

Reply:
after compiled, `apache_inlong_manager.sql` in `inlong-manager/sql/`,`apache_inlong_audit.sql` in `inlong-audit/sql/`